lib/chat_api/slack/event.ex

```diff
@@ -0,0 +1,432 @@
+defmodule ChatApi.Slack.Event do
```

Comment:
(just setting up new module to hold all our slack event handlers... this code is basically all copied from the slack controller)